app/src/main/java/com/example/next_show/data/RecommendationClient.java

```diff
@@ -104,18 +103,15 @@ public void onFailure(Call<List<com.uwetrottmann.trakt5.entities.Show>> call, Th
         }
     }
 
-    public static List<Show> getGenreMatch(List<Show> shows, User user){
-        // get proper user genres -> there is query that does this for Trakt
-        List<String> favoriteGenres = user.getFaveGenres();
-
+    public static List<Show> getGenreMatch(List<Show> shows, List<String> compareGenres){
         // init return list
         List<Show> genreMatched = new ArrayList<>();
 
         for (Show s: shows) {
             List<String> showGenres = s.getGenres();
 
-            // if any genre(s) match the favorite ones
-            if (!Collections.disjoint(showGenres, favoriteGenres)) {
+            // if any genre(s) match the comparison ones
+            if (!Collections.disjoint(showGenres, compareGenres)) {
                 genreMatched.add(s);
             }
         }
```

Comment:
I think this could maybe live in the show filter now, right?  You just need something that will return you the genres to match against and then this is basically just filtering on the shows